### PR TITLE
added link to huggingface to the doc

### LINKED ISSUE #718: huggingface server not mentioned in the readme page

### PR DESCRIPTION
**Pull request by saeid93**
fixes #718

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +93,7 @@ MLServer to start serving your machine learning models.
 - [Serving a `tempo` pipeline](./docs/examples/tempo/README.md)
 - [Serving a custom model](./docs/examples/custom/README.md)
 - [Serving an `alibi-detect` model](./docs/examples/alibi-detect/README.md)
+- [Serving a `HuggingFace` model](./docs/examples/huggingface/README.m)
```
Comment (adriangonz) [inline README.md:96]: Is there a missing `d` in `.md`? Not sure if Sphinx is happy with that.

@saeid93 would you be able to test it out on the RTD preview for this PR?

https://mlserver--719.org.readthedocs.build/en/719/

**Review by adriangonz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (saeid93) [reply README.md]: Please check now. I think it should be fine. Testing locally looks fine.

**Review by adriangonz — APPROVED** [review]
Awesome! Thanks for addressing the changes @saeid93! :+1:

Once the tests are green, we should be good to go ahead.

**Comment by saeid93** [comment]
@adriangonz is it all good now? please let me know if anything else is needed.

**Pull request merged**